utils/fixed_decimal/src/decimal.rs

```diff
@@ -49,6 +49,14 @@ const_assert!(core::mem::size_of::<usize>() >= core::mem::size_of::<u16>());
 ///
 /// dec.multiply_pow10(-2);
 /// assert_eq!("2.50", dec.to_string());
+///
+/// // IEEE 754 for floating point defines the sign bit separate
```

Comment:
Nit: Put this example in the docs for `from_f64` to demonstrate that we retain the sign of the `-0.0` literal.

Reply:
Created the example at https://github.com/unicode-org/icu4x/pull/1807/files#diff-5851ca6096651367eced534b6a8b82f133009b5c71dc0e5963bd3a8fdaa7da5bR960!

I wasn't sure if you meant to remove the original example from the top-layer, so both are part of this PR